native/src/test/java/io/ballerina/stdlib/http/transport/http2/frameleveltests/Http2ChannelCloseAfterConnectionEvictionAfterTcpServerGoAwayScenarioTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```
Shall we update the license header. Refer `License Header for WSO2 code` mail thread.